helm-charts/splunk-otel-collector/templates/daemonset.yaml

```
@@ -269,6 +269,8 @@ spec:
           {{- include "splunk-otel-collector.securityContext" (dict "isWindows" .Values.isWindows "securityContext" $agent.securityContext "setRunAsUser" true) | nindent 10 }}
         {{- end }}
         env:
+          - name: GOMEMLIMIT
+            value: {{ include "splunk-otel-collector.convertMemToMib" $agent.resources.limits.memory | int64 }}
```

The test might be failing here due to a quotation or some other issue, likely also affects deployment-gateway.yaml.
```suggestion
            value: "{{ include "splunk-otel-collector.convertMemToMib" $agent.resources.limits.memory | int64 }}"
```